src/modules/stillages/stillages.service.ts

```diff
@@ -141,3 +145,8 @@ export class StillagesService {
     `;
   }
 }
+
+function convertTimestampToDate(timestamp: Date) {
+  const formattedDate = timestamp.toISOString().slice(0, 10); // Extract YYYY-MM-DD
```

Comment:
method toISOString() can [cause some bugs](https://stackoverflow.com/questions/43670062/get-iso-string-from-a-date-without-time-zone). Method setHours(0, 0, 0, 0) should be used here